controllers/HomeController.php

```diff
@@ -16,7 +17,30 @@ class HomeController extends Controller
      */
     public function viewHome() {
         echo $this->twig->render('home.html.twig', [
-            'variable' => 'Twig'
+            'title' => 'Blog de David - Accueil'
         ]);
     }
+
+    /**
+     * Send an email with the form
+     *
+     * @param  string $to
+     * @param  string $name
+     * @param  string $message
+     *
+     * @return void
+     */
+    public function sendEmail($to, $name, $message) {
```

Comment:
accolade sur la ligne d'après et typer les paramètres (string $to, string $name, string $message)